CMakeLists.txt

```diff
@@ -210,6 +210,17 @@ if(ENABLE_SHARED)
     )
     # Set the SOVERSION property of the wiredtiger library, so we can export the appropriately versioned symlinks.
     set_target_properties(wiredtiger_shared PROPERTIES SOVERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
+
+    # For MacOS builds we need to generate a dSYM bundle that contains the debug symbols for each 
+    # the WiredTiger library.
+    if (WT_DARWIN AND HAVE_DIAGNOSTIC)
+        add_custom_command(
+            TARGET wiredtiger_shared POST_BUILD
+            COMMAND dsymutil libwiredtiger.${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}.dylib
```

Comment:
How does this end up working in practice? Do I need to load two dSYMs, one for the library and one for the binary?

Reply:
In practice LLDB will look for the corresponding dSYM files on the machine. So after my first test of generating the dSYM bundle for the shared library the subsequent tests would automatically find that same bundle. So this is still necessary for having the correct symbols for that commit of wiredtiger.

Reply:
Okay. It sounds like we might need to manually load two bundles when debugging with downloaded artifacts then?

(That's not a problem and we already have to do something similar with GDB's `set solib-search-path`, I just want to confirm my understanding.)

Reply:
Yeah, I believe so.